app/models/user.rb

```diff
@@ -58,7 +58,11 @@ class User < ApplicationRecord
   validate :has_at_least_one_office
   validate :has_at_least_one_age_range, if: :require_volunteer_profile_attributes?
 
-  scope :volunteers, -> { where(role: 'volunteer').or(where(role: 'coordinator')) }
+  scope :social_workers, -> { where(role: 'social_worker') }
```

Comment:
Would probably make constants for these roles. Eg:

```
COORDINATOR   = 'coordinator'
SOCIAL_WORKER = 'social_worker'
VOLUNTEER     = 'volunteer'

scope :coordinators, -> { where(role: COORDINATOR) }
scope :social_workers, -> { where(role: SOCIAL_WORKER) }
scope :volunteers, -> {  where(role: VOLUNTEER) }
```

Note the constants don't need `freeze` because they are string literals and we're using the `frozen_string_literal: true` pragma.